docker/run_rocker.bash

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
```

Comment:
Can be deleted

Reply:
The shebang? This line is present in all the shell scripts. If it is run from a different shell, like sh, it tells sh to use bash. I would leave it in.

Reply:
No, I meant the entire file, or are we keeping rocker stuff in for now?

Reply:
Oh! Didn't see the title of that file. Yes should be removed.
Removed in 4ee2f1c